src/RapidezServiceProvider.php

```diff
@@ -60,6 +61,12 @@ protected function bootCommands(): self
             InstallTestsCommand::class,
         ]);
 
+        Event::listen(IndexBeforeEvent::class, function ($event) {
+            if (config('rapidez.autocomplete_categories')) {
+                $event->context->call('rapidez:index:categories');
+            }
+        });
+
```

Comment:
Do we want this to be dependent on the config setting? It feels to me like this could easily become spaghetti in a codebase with very custom frontend. The fact that it also auto-runs the indexer doesn't seem like it makes much sense from an outside perspective.

Reply:
Nope, you can remove the config. Let's just always index the categories 👍🏻